skiko/src/awtMain/cpp/windows/DCompLibrary.hh

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <dcomp.h>
+
+namespace DCompLibrary {
+    HINSTANCE getLibrary() {
+        static HINSTANCE library = LoadLibrary("dcomp.dll");
```

Comment:
You must split header and implementation. Currently it will create a copy on each include occurrence

Reply:
Done